components/camel-azure/camel-azure-storage-datalake/src/main/java/org/apache/camel/component/azure/storage/datalake/DataLakeConstants.java

```diff
@@ -21,192 +21,192 @@
 
-    @Metadata(label = "from user",
+    @Metadata(label = "producer",
               description = "Defines options available to configure the behavior of a call to listFileSystemsSegment on a DataLakeServiceAsyncClient object. Null may be passed.",
               javaType = "ListFileSystemsOptions")
     public static final String LIST_FILESYSTEMS_OPTIONS = HEADER_PREFIX + "ListFileSystemsOptions";
-    @Metadata(label = "from user", description = "An optional timeout value beyond which a RuntimeException will be raised.",
+    @Metadata(label = "producer", description = "An optional timeout value beyond which a RuntimeException will be raised.",
               javaType = "Duration")
     public static final String TIMEOUT = HEADER_PREFIX + "Timeout";
-    @Metadata(label = "from user",
+    @Metadata(label = "producer",
               description = "Specify the producer operation to execute. Different operations allowed are shown below.",
               javaType = "org.apache.camel.component.azure.storage.datalake.DataLakeOperationsDefinition")
     public static final String DATALAKE_OPERATION = HEADER_PREFIX + "Operation";
-    @Metadata(label = "from user",
+    @Metadata(label = "producer",
               description = "Name of the file system in azure datalake on which operation is to be performed. Please make sure that filesystem name is all lowercase.",
               javaType = "String")
     public static final String FILESYSTEM_NAME = HEADER_PREFIX + "FileSystemName";
-    @Metadata(label = "from user",
+    @Metadata(label = "producer",
               description = "Name of the directory in azure datalake on which operation is to be performed.",
               javaType = "String")
     public static final String DIRECTORY_NAME = HEADER_PREFIX + "DirectoryName";
-    @Metadata(label = "from user", description = "Name of the file in azure datalake on which operation is to be performed.",
+    @Metadata(label = "producer", description = "Name of the file in azure datalake on which operation is to be performed.",
               javaType = "String")
     public static final String FILE_NAME = HEADER_PREFIX + "FileName";
     @Metadata(label = "from both", description = "The metadata to associate with the file.",
               javaType = "Map<String, String>")
     public static final String METADATA = HEADER_PREFIX + "Metadata";
-    @Metadata(label = "from user",
+    @Metadata(label = "producer",
               description = "Defines options available to configure the behavior of a call to listFileSystemsSegment on a DataLakeServiceAsyncClient object.",
               javaType = "PublicAccessType")
     public static final String PUBLIC_ACCESS_TYPE = HEADER_PREFIX + "PublicAccessType";
-    @Metadata(label = "from component", description = "Non parsed http headers that can be used by the user.",
+    @Metadata(label = "consume", description = "Non parsed http headers that can be used by the user.",
```

Comment:
I guess you meant `consumer`, didn't you?

Reply:
Yes I do.

Reply:
Let me fix that.